runbotics/runbotics-orchestrator-ui/src/components/HistoryTable/HistoryTable.content.tsx

```diff
@@ -45,14 +48,18 @@ const HistoryTable = forwardRef<any, HistoryTableProps>(({
     const { page: processInstancePage, loadingPage } = processInstances.all;
     const [panelInfoState, setPanelInfoState] = useState<PanelInfoState>({ show: false });
     const processInstanceColumns = useProcessInstanceColumns();
-    
-
     const history = useHistory();
     const query = useQuery();
     const pageFromUrl = query.get('page');
     const [page, setPage] = useState(pageFromUrl ? parseInt(pageFromUrl, 10) : 0);
     const pageSizeFromUrl = query.get('pageSize');
     const [pageSize, setPageSize] = useState(pageSizeFromUrl ? parseInt(pageSizeFromUrl, 10) : DefaultPageSize.TABLE);
+    const hasProcessInstanceEventReadAccess = useFeatureKey([FeatureKey.PROCESS_INSTANCE_EVENT_READ]);
+    const { user } = useAuth()
+
+    const accessedColumns = processInstanceColumns.filter((column) =>
```

Comment:
imo this logic could be done in useProcessInstanceColumns

Reply:
done